pkg/uiserver/uiserver.go

```diff
@@ -53,6 +61,63 @@ func RewriteAssets(fs http.FileSystem, cfg *config.Config, updater UpdateContent
 
 	rewrite("/index.html")
 	rewrite("/diagnoseReport.html")
+
+	overrideDistroAssetsRes(fs, cfg, updater)
+}
+
+func overrideDistroAssetsRes(fs http.FileSystem, cfg *config.Config, updater UpdateContentFunc) {
+	exePath, err := os.Executable()
+	if err != nil {
+		log.Fatal("Failed to get work dir", zap.Error(err))
+	}
+
+	distroResDir := path.Join(path.Dir(exePath), distro.DistroResFolderName)
+	info, err := os.Stat(distroResDir)
+	if err != nil || !info.IsDir() {
+		// just ignore
+		return
+	}
+
+	override := func(assetName string) {
```

Comment:
How about moving this function out? It would be better to be not a lambda function.